### ISSUE TITLE
[Accessibility]: Search box screen reader issues

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#34732]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34188

### Problem description

- [ ] Running a search doesn't announce anything (should announce to the user that a search is being executed)
- [ ] When activating the "copy query" button, "Copied" is read twice (should only read it once)

### Expected behavior

Fixed as described above

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
